tools/docker/builder/rdkb/toolchain/bitbake.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2020 Tomer Eliyahu (Intel)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}"; pwd)"
+rootdir="${scriptdir%/*/*/*/*/*}"
+
+. ${rootdir}/tools/docker/functions.sh
+
+usage() {
+    echo "usage: $(basename $0) [-hrv]"
+    echo "  options:"
+    echo "      -h|--help - show this help menu"
+    echo "      -r|--rdkb-root - path to rdkb root (default is PWD)"
+    echo "      -i|--build-image - Only build image"
+    echo "      -v|--verbose - verbosity on"
+    echo ""
+}
+
+build_image() {
+    docker build --tag rdkb-builder "$scriptdir/"
+}
+
+main() {
+
+    OPTS=`getopt -o 'hir:v' --long help,image,rdkb-path:,verbose -n 'parse-options' -- "$@"`
+
+    if [ $? != 0 ] ; then err "Failed parsing options." >&2 ; usage; exit 1 ; fi
+
+    eval set -- "$OPTS"
+
+    while true; do
+        case "$1" in
+            -h | --help)               usage; exit 0; shift ;;
+            -i | --image)              build_image; exit 0; shift ;;
+            -r | --rdkb-root)          RDKB_ROOT="$2"; shift; shift ;;
+            -v | --verbose)            VERBOSE=true; shift ;;
+            -- ) shift; break ;;
+            * ) err "unsupported argument $1"; usage; exit 1 ;;
+        esac
+    done
+
+    build_image
+    # Default docker arguments
+    docker_args="\
+     --workdir=$PWD \
```

Comment:
if $PWD contains spaces, it breaks (the same is true for all the other options inside `docker_args`).
There is no easy way to avoid it without using quotes/eval, which is why having command args inside a string can be considered an anti-pattern.
You could use a bash array instead, but the easiest is probably to give the args to the docker command directly, since `docker_args` is not used elsewhere.